wildvvad/utils/model.py

```diff
@@ -0,0 +1,119 @@
+import keras
+from keras.layers import Bidirectional, Dense, Flatten
+from keras.layers.normalization import BatchNormalization
+from keras.layers.wrappers import TimeDistributed
+from keras.models import Sequential
+
+import tensorflow as tf
+
+
+class LAND_LSTM_Model:
+    """
+        Build Bidirectional LSTM Model based on the WildVVAD description.
+        Each video sample is mapped onto a sequence of vectors,
+        where each vector contains the frontal 3D coordinates of the 68 landmarks.
+        This sequence is then fed into a bidirectional LSTM, [16], in which fully
+        connected layers share the parameters across time. We employ
+        ReLu activations and add batch normalization layers [17]
+        between bidirectional LSTM layers.
+    """
+
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def build_land_lstm_tuner(hp, input_shape, num_td_dense_layers=1,
```

Comment:
What is the difference here - why is it different for the tuner? try to keep the code DRY

Reply:
@adriandavidauer Yes, can be optimized and will be done